scripts/ghutils.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# This script contains functions for interacting with github.
+# At one stage it contained code for setting github statuses,
```

Comment:
This comment seems confusing, if I understand correctly, it's saying a file of a different name used to exist and contained code that is no longer required and has not been used for a while. I don't think it adds anything useful.